assets/js/wcmp-admin.js

```diff
@@ -3,15 +3,22 @@
  *
  * @property {Object} wcmp.actions
  * @property {{export: String, add_shipments: String, add_return: String, get_labels: String}} wcmp.actions
+ * @property {String} wcmp.api_url - The API Url we use in MyParcel requests.
  * @property {String} wcmp.ajax_url
- * @property {String} wcmp.nonce
+ * @property {String} wcmp.ask_for_print_position
+ * @property {Object} wcmp.bulk_actions
+ * @property {{export: String, print: String, export_print: String}} wcmp.bulk_actions
  * @property {String} wcmp.download_display
- * @property {String} wcmp.offset
+ * @property {String} wcmp.nonce
  * @property {String} wcmp.offset_icon
  */
 
 // eslint-disable-next-line max-lines-per-function
```

Comment:
Als de code geminified is, zal `//` voor problemen zorgen 

```suggestion
// eslint-disable-next-line max-lines-per-function
```
/* eslint-disable-next-line max-lines-per-function */